### PR TITLE
[BEAM-2855] Add queries 1 and 2 to the Python Nexmark suite

### PR DESCRIPTION
**Pull request by mariapython**

### CODE REVIEW

**Comment by mariapython** [comment]
R: @charlesccychen

File: sdks/python/apache_beam/testing/benchmarks/nexmark/nexmark_launcher.py [hunk 1]
```diff
@@ -83,6 +85,8 @@ def __init__(self):
     self.topic_name = self.args.topic_name + self.uuid
     self.subscription_name = self.args.subscription_name + self.uuid
 
+    print('topic_name:', self.topic_name)
```
Comment (charlesccychen) [inline sdks/python/apache_beam/testing/benchmarks/nexmark/nexmark_launcher.py]: Was this for debugging?

File: sdks/python/apache_beam/testing/benchmarks/nexmark/queries/query1.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  logging.debug(elm)
+  return elm
+
+
+def load(raw_events, query_args=None):
+  return (raw_events
+          | 'ParseEventFn' >> beam.ParDo(ParseEventFn())
+          | 'FilterInBids' >> beam.Filter(
+              lambda event: isinstance(event, nexmark_model.Bid))
+          | 'ConvertToEurop' >> beam.Map(
```
Comment (charlesccychen) [inline sdks/python/apache_beam/testing/benchmarks/nexmark/queries/query1.py]: Typo? "Europ"

File: sdks/python/apache_beam/testing/benchmarks/nexmark/queries/query2.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+import logging
+
+import apache_beam as beam
+from apache_beam.testing.benchmarks.nexmark.models import nexmark_model
+from apache_beam.testing.benchmarks.nexmark.nexmark_util import ParseEventFn
+
+
+def display(elm):
+  logging.debug(elm)
+  return elm
```
Comment (charlesccychen) [inline sdks/python/apache_beam/testing/benchmarks/nexmark/queries/query2.py]: Should we refactor this into nexmark_util?

File: sdks/python/apache_beam/testing/benchmarks/nexmark/nexmark_launcher.py (hunk 1 above)
Comment (mariapython) [reply sdks/python/apache_beam/testing/benchmarks/nexmark/nexmark_launcher.py]: Oops. I didn't pop some corrections I had made. Done!

File: sdks/python/apache_beam/testing/benchmarks/nexmark/queries/query1.py (hunk 2 above)
Comment (mariapython) [reply sdks/python/apache_beam/testing/benchmarks/nexmark/queries/query1.py]: Same for this one. Done!

File: sdks/python/apache_beam/testing/benchmarks/nexmark/queries/query2.py (hunk 3 above)
Comment (mariapython) [reply sdks/python/apache_beam/testing/benchmarks/nexmark/queries/query2.py]: Good idea, thanks. Done.

**Comment by charlesccychen** [comment]
Thanks, LGTM.

**Pull request merged**